packages/site/src/utils/localStorage.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Get a local storage key.
+ *
+ * @param key - The local storage key to access.
+ * @returns The value stored at the key provided if the key exists.
+ */
+export const getLocalStorage = (key: string) => {
+  const { localStorage: ls } = window;
+
+  if (ls !== null) {
+    const data = ls.getItem(key);
+    return data;
+  }
+  return null;
+};
+
+/**
+ * Set a value to local storage at a certain key.
+ *
+ * @param key - The local storage key to set.
+ * @param value - The value to set.
+ */
+export const setLocalStorage = (key: string, value: string) => {
+  const { localStorage: ls } = window;
+
+  if (ls !== null) {
+    ls.setItem(key, value);
+  }
+};
```

Comment:
There is no way to know if this failed, because it doesn't return anything. Should we throw an error if `ls` doesn't exist?